doc/install.rst

```diff
@@ -0,0 +1,148 @@
+ along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
+
+.. Edited September 27, 2020 - [USA]TechDude
+
+Install |ops| OpenShot
+-----------------------
+
+OpenShot™ is available for the following 64-bit operating systems: Microsoft 
+Windows®, Linux® (most distributions), and Apple® macOS®.
+
+|win| Windows®
+^^^^^^^^^^^^^^
+
+Download the Windows installer executable from the official OpenShot |Link|_.
+
+.. image:: images/install-win1.png
+
+Choose to save the file to your system so you can launch the installer after 
+the download completes.
+
+.. image:: images/install-win2.png
+
+Once downloaded, launch the EXE file, and follow the directions on screen. 
+Choose the language that you want to use if it is not already highlighted in 
+the pull-down menu. 
+
+.. image:: images/install-win3.png
+
+Read the License Agreement and select the radio button in front of "I accept 
+the agreement" if you agree to the terms.
+
+.. image:: images/install-win4.png
+
+Choose where to install OpenShot or use the default path provided.
+
+.. image:: images/install-win5.png
+
+Select any additional shortcuts that you want setup to create when it installs 
+the application.
+
+.. image:: images/install-win6.png
+
+Review the options for installing OpenShot.  Click the back button to make 
+changes or click Install to install OpenShot on your system.
+
+.. image:: images/install-win7.png
+
+When Setup completes, you can choose to launch the program by leaving the check 
+in the box next to "Launch OpenShot Video Editor" and clicking "Finish".
+
+.. image:: images/install-win8.png
+
+Setup created an icon in your Start Menu from where you can launch the program.
+
+.. image:: images/install-win9.png
+
+|lin| Linux®
+^^^^^^^^^^^^
+
+Download the Linux AppImage from the official OpenShot |Link|_.  
+
+.. image:: images/install-lin1.png
+
+Choose to save the file to your computer.
+
+.. image:: images/install-lin2.png
+
+Once downloaded, move the file to the folder where it will live.  
+
+.. image:: images/install-lin3.png
+
+Right-click the AppImage file, click Properties then mark the file as 
+**Executable**.  
+
+.. image:: images/install-lin4.png
+
+Double-click the AppImage file to launch the application.
+
+|mac| macOS®
+^^^^^^^^^^^^
+
+Download the disk image (DMG) file from the official OpenShot |Link|_.  
+
+.. image:: images/install-mac1.png
+
+Choose to save the file to your system so you can launch the installer after 
+the download completes.
+
+.. image:: images/install-mac2.png
+
+Double-click the DMG file to open the image.  This allows you to drag the 
+application to your Applications folder.
+
+.. image:: images/install-mac3.png
+
+Locate the icon in your Applications folder and double-click it to launch 
+OpenShot.
+
+.. image:: images/install-mac4.png
+
+If you receive an error message that states: 
+
+.. image:: images/error-mac1.png
+
+you can correct this issue. Apple macOS needs permission to run the file that 
```

Comment:
This is not needed with our daily builds anymore. No settings need to be changed to install and run on macOS now.